src/i18n/en/nav.ts

```diff
@@ -78,6 +78,22 @@ export default [
 		slug: 'guides/server-side-rendering',
 		key: 'guides/server-side-rendering',
 	},
+	{ text: 'Authoring Content', slug: 'guides/content', key: 'guides/content' },
+	{
+		text: 'Content Collections',
+		slug: 'guides/content-collections',
+		key: 'guides/content-collections',
+	},
+	{ text: 'Connecting a CMS', slug: 'guides/cms', key: 'guides/cms' },
+	{ text: 'Backend as a Service', slug: 'guides/baas', key: 'guides/baas' },
+	{ text: 'Images', slug: 'guides/images', key: 'guides/images' },
+	{ text: 'Fonts', slug: 'guides/fonts', key: 'guides/fonts' },
+	{
+		text: 'Sharing State',
+		slug: 'core-concepts/sharing-state',
+		key: 'core-concepts/sharing-state',
+	},
+	{ text: 'RSS', slug: 'guides/rss', key: 'guides/rss' },
```

Comment:
This whole edit is messed up, since it was probably from the original PR before we had recipes.

Make sure it's only adding the backend section to recipes, and nothing else is changed.